test/CMakeLists.txt

```diff
@@ -14,6 +14,12 @@ function(swift_configure_lit_site_cfg source_path destination_path installed_nam
     set(LIT_SWIFTLIB_DIR ${SWIFTLIB_DIR})
   endif ()
 
+  if(SWIFT_HAVE_LIBXML)
```

Comment:
Can you use a different variable name here for the configure? I am nervous about people getting confused. Changing the name would make me more confortable.

Reply:
Ugh, naming is hard.  I think that an alternate name is a great idea, but I don't really have any good suggestions.  Mind providing some suggestions?

Reply:
LIT_SWIFT_HAVE_LIBXML

Reply:
BTW, `lit.site.cfg` has a number of the variables with the same value.  I think that the problem is that the value being saved here is not guaranteed to be normalized :-(.